packages/core/src/browser/style/hover-service.css

```diff
@@ -29,7 +29,12 @@
   border: 1px solid var(--theia-editorHoverWidget-border);
   padding: var(--theia-ui-padding);
   max-width: var(--theia-hover-max-width);
-  z-index: 1000;
+}
+
+/* user agent style-sheet has overflow:hidden, so pointer is not visible but scroll is */
```

Comment:
nit: For me on Chrome, the user agent stylesheet has `overflow:auto` and not `overflow:hidden`, maybe we should change the comment to simply say we want to overwrite the default user agent stylesheet.

Reply:
```suggestion
/* ensure overflow is visible as user agent styles might have different defaults */
```